src/platform/FailSafeContext.cpp

```diff
@@ -42,21 +70,25 @@ void FailSafeContext::FailSafeTimerExpired()
     event.FailSafeTimerExpired.UpdateNocCommandHasBeenInvoked = mUpdateNocCommandHasBeenInvoked;
     CHIP_ERROR status                                         = PlatformMgr().PostEvent(&event);
 
-    mFailSafeArmed                  = false;
```

Comment:
We should still be setting `mFailSafeArmed` to false here, so the logic here and in the platform manager boot code will be the same.  Also, so if a command comes in at this point that needs an armed fail-safe it will fail...